vcd/resource_vcd_rde_test.go

```diff
@@ -507,21 +518,21 @@ func addRightsToTenantUser(t *testing.T, vcdClient *VCDClient, vendor, nss strin
 	}
 	err = role.AddRights(rightsToAdd)
 	if err != nil {
-		t.Errorf("could not add rights '%v' to role '%s'", rightsToAdd, role.GlobalRole.Name)
+		t.Fatalf("could not add rights '%v' to role '%s'", rightsToAdd, role.GlobalRole.Name)
```

Comment:
Curious, why does it have to be fatal and error is not enough?

Reply:
The Errorf marks the test as failed but continues execution, which can produce some nil pointers and undesired behavior. On the other hand Fatalf marks the test as failed and stops the execution of the test immediately.